plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -0,0 +1,59 @@
+import { Link } from "react-router";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Breadcrumb from "#SRC/js/components/Breadcrumb";
+import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
+import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
+
+export default function Breadcrumbs({ item, children, renderStates }) {
+  function getBreadcrumb(item, id = "", name = "") {
+    const isDetailPage = [...item.path, item.name].join(".") === id;
+    const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
+
+    return (
+      <Breadcrumb key={id} title="Jobs">
+        <BreadcrumbTextContent>
+          <Link to={link}>{name === "" ? "Jobs" : name}</Link>
+        </BreadcrumbTextContent>
+
+        {isDetailPage ? renderStates(item) : null}
```

Comment:
I have mixed feelings about that one. So this is going to be a very special component for Jobs Breadcrumbs?

Reply:
to clarify, having the function `renderStates` feels uncomfortable

Reply:
Please propose an alternative. I need a way to not handle that data within this component to a) be able to also handle the job overview page and b) comply with @juliangieseke feedback

Reply:
My suggestion would be a separate component for the Overview page.

Reply:
Okay, so we would basically use the status and schedule component from within here and do a copy without them for the overview? I feel like we should discuss this at least with @juliangieseke because the result would be very very similar to what I had before coming to work yesterday, so I would like to double down to the decision to not waste any more time.